ros_ign_bridge/CMakeLists.txt

```diff
@@ -18,14 +18,22 @@ find_package(rosgraph_msgs REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(std_msgs REQUIRED)
 
-# Citadel
 find_package(ignition-transport8 QUIET)
+find_package(ignition-transport9 QUIET)
```

Comment:
In order to avoid problems in case both `transport8` and `transport9` are available, can we move the find call for `transport9` into a block where `transport8` is not found?

Reply:
[Done](https://github.com/ignitionrobotics/ros_ign/pull/103/commits/b40e4ff37b2364c91d5a18bbd6c61a734da750af)